### ISSUE TITLE
Automatic departure code clears departures on crossing runways

### ISSUE THREAD

**Issue by scmmmh** [openscope/openscope#1827]
Browser: Firefox 89

Airport: KSFO

Approximate play time before issue: 0sec

#### Issue Description

The new automatic departure code clears flights for departure that are departing on crossing runways.

#### Steps to reproduce

- Load KSFO
- Watch parallel departures on 28L/28R/1L/1R

**Comment by erikquinn** [comment]
![image](https://user-images.githubusercontent.com/5103735/121272058-02f09100-c893-11eb-8394-8e862f63acb8.png)

SFO does not typically depart both the 28s and 1s while landing the 28s... they would land 28s and depart the 1s. Full breakdown of airport acceptance rates in various configurations (source: FAA).

The solution to this _exact_ situation is to change SFO to only depart the 1s (and maybe just 1L) to prevent conflicts since the system isn't smart enough to separate them from each other. If you have any other airports this happens at, we can open other issues for them (I spot LSZH as the only other one with this issue).

**Comment by scmmmh** [comment]
This fixes this specific issue, but not the more general issue that the auto-departure code does not check conflicting runways. For example, planes will be depart even if another plane is just landing on the conflicting runway. Should I open a separate issue for that?

**Comment by erikquinn** [comment]
Yes, taking that into a consideration would be an enhancement issue, and you're welcome to open it.
